http_check/datadog_checks/http_check/http_check.py

```diff
@@ -52,12 +52,6 @@ def __init__(self, name, init_config, instances):
         if not self.ca_certs:
             self.ca_certs = get_ca_certs_path()
 
-        self.HTTP_CONFIG_REMAPPER['ca_certs']['default'] = self.ca_certs
```

Comment:
Setting the `default` here will force `requests` to use those instead of the one they package with `certifi` already.